routes/ideaAPI.js

```diff
@@ -16,12 +16,16 @@ var router = require('express').Router();
 //		deadline:Date
 //		hrRequirement:String
 //		relatedAssets:[String] (Asset)
+//response:
+//{idea:IdeaEntity}
 router.post('/add',authenticator.authenticate,ideaController.createIdea);
 
 //query parameters
 //	required:
 //		id:String (idea)
-router.get('/id/:id',ideaController.getIdeaById);
+//response:
+//{idea:IdeaEntity}
+router.get('/:id',ideaController.getIdeaById);
```

Comment:
why do you change back? Interface should be separated well.  You mixed the "/actiond_name" and "/:parameter" pattern.